nwb_conversion_tools/BaseDataInterface.py

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
If this file contains non-base classes, can we not call it "Base"?

Reply:
Fair; should we use the structure Alessio uses, then? Have one file called Base, and then a folder of other interfaces filled with a separate file defining the name of that subclass?

Reply:
maybe down the line we could do that, but I don't think we really have enough to warrant that level of structure yet

Reply:
All non-base subclasses have been separated out into appropriately named files.